sympy/utilities/tests/test_wester.py

```diff
@@ -2945,14 +2944,13 @@ def test_Y5_Y6():
     t = symbols('t', positive=True)
     s = symbols('s')
     y = Function('y')
+    Y = Function('Y')
     F, _, _ = laplace_transform(diff(y(t), t, 2)
                                 + y(t)
                                 - 4*(Heaviside(t - 1)
                                 - Heaviside(t - 2)), t, s)
-    # Laplace transform for diff() not calculated
-    # https://github.com/sympy/sympy/issues/7176
-    assert (F == s**2*LaplaceTransform(y(t), t, s) - s
-            + LaplaceTransform(y(t), t, s) - 4*exp(-s)/s + 4*exp(-2*s)/s)
+    G = laplace_transform_replace_ic(F, y, Y, t, s, [1, 0])
+    assert (G == s**2*Y(s) - s + Y(s) - 4*exp(-s)/s + 4*exp(-2*s)/s)
```

Comment:
I wonder if it would be better to extend the signature of `laplace_transform` rather than have a separate function for this. I can see the argument both ways but I expect that a lot of users would want `laplace_transform` to directly convert `L{y(t)}` to `Y(t)` rather than call a separate function afterwards. I haven't really considered what a good way of allowing users to specify that would be...

Reply:
Or maybe there could be a separate function called e.g. `laplace_transform_ode` that is specifically for applying the Laplace transform to ODEs.

Reply:
Perhaps this particular part of the PR could be deferred to another PR so we can get the rest in and consider these details later.

Reply:
If all of you agree with the project plan I outlined in the reply to @asmeurer below, we could do this in step 4 later.